cmd/prometheus/main.go

```diff
@@ -139,10 +139,12 @@ var (
 	agentOnlyFlags, serverOnlyFlags []string
 )
 
+var DefaultRegistry = prometheus.NewRegistry()
```

Comment:
I would just call it `registry`. No reason to export anything in `main.go`, and the word `default` seems out of place as it is the intent to _not_ use the default registry.